src/ExpectedMark.js

```diff
@@ -31,14 +31,26 @@ export default class ExpectedMark extends UXBase {
 		return _expectedMarks[name];
 	}
 
-	static clearExpectedMarksMap() {
+	static destroy(name) {
+		if (typeof window.performance !== 'undefined') {
+			window.performance.clearMarks(name);
+		}
+		if (name) {
+			delete _expectedMarks[name];
+			return;
+		}
 		_expectedMarks = {};
 	}
 
 	// registers zone callback
-	onComplete(onMark) {
+	addOnMarkListener(onMark) {
```

Comment:
renamed this method to match 'listener' functions in other JS APIs and the `onMarkListeners` terminology used internally

Reply:
I went back and forth renaming this, we also have this ticket describing the issue: https://meetup.atlassian.net/browse/WP-839

Sadaf and I discussed this and decided to keep this ticket but not make it urgent, please chime in.